src/web/app/pages-student/student-course-details-page/student-course-details-page.component.spec.ts

```diff
@@ -105,4 +111,388 @@ describe('StudentCourseDetailsPageComponent', () => {
     fixture.detectChanges();
     expect(fixture).toMatchSnapshot();
   });
+
+  it('should sort teammate profiles by name after clicking on "name"', () => {
+    const student: Student = {
```

Comment:
A lot of these constants are duplicated across the test cases. Please shift them outside of the individual testcases so that they can be shared among all the tests.

Reply:
Hi, I tried to update my local repo and commit again but it failed E2E test. Then I rolled back to outdated version and shift duplicated codes. Is it OK if other codes are outdated?

Reply:
Do keep this PR up to date with the `master` branch; E2E tests may sometimes need multiple runs to pass, we will help to monitor the E2E test runs and will notify you if any changes are needed.